config/default/views.view.find_a_campground_or_recreation_site.yml

```diff
@@ -7,11 +7,6 @@ dependencies:
     - node.type.campground_directory_record
     - taxonomy.vocabulary.region
     - taxonomy.vocabulary.site_type
-  content:
-    - 'taxonomy_term:site_type:464b2caa-cf1d-46da-800f-0cace9d7a7f6'
-    - 'taxonomy_term:site_type:d7b69697-152b-4ef3-b853-2d6ad1340491'
-    - 'taxonomy_term:site_type:f2f63b8d-2f0e-469b-a64b-fe1e0922cdc6'
-    - 'taxonomy_term:site_type:f7783847-bf85-45a1-8f05-21e8240a4759'
```

Comment:
I'm not sure about this change. Need farther investigation.

Reply:
Please compare the view result with the old version

Reply:
I reconfigured 'Find a campground or recreation site' (find_a_campground_or_recreation_site) view based on D7 configuration.